src/main/java/org/fundacionjala/coding/carlos/highestandlowest/CalculateNumbers.java

```diff
@@ -0,0 +1,28 @@
+package org.fundacionjala.coding.carlos.highestandlowest;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+
+public class CalculateNumbers {
+
+    public List<Integer> colectNumbers(String numbers) {
+        List<Integer> numbersList = new ArrayList<>();
+        String[] splitNumbers = numbers.split(" ");
+        for (int i=0; i<splitNumbers.length; i++){
```

Comment:
Please consider using a `foreach` statement. Something like:
```java
for(String number : splitNumbers) {
  numberList.add(Integer.parseInt(number));
}
```

Reply:
ok, I apply it now thanks